test/integration/guest_user_test.rb

```diff
@@ -7,7 +7,7 @@ class GuestUserTest < ActionDispatch::IntegrationTest
 
   def setup
     @item1 = Item.create(name: 'espresso', cost: 9000)
-    @category1 = item1.categories.create(name: 'hot beverages')
+    @category1 = item1.categories.create(name: 'Hot Beverages')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.